### ISSUE TITLE
Eto.Platform.iOS cannot be installed

### ISSUE THREAD

**Issue by Berrysoft** [picoe/Eto#1438]
I created a Xamarin.iOS project and tried to install Eto.Platform.iOS 2.3.0 (which is the latest version, why that old?), and failed because the project targets xamarin10 but the package targets monotouch. Did I do something wrong?

**Comment by cwensley** [comment]
Yes, the iOS platform is out of date and has unfortunately not been maintained.

For mobile platforms, I recommend to use something like Xamarin.Forms.  There has not been much interest in getting Eto.Forms to support mobile thus far so effort hasn't been focussed there.
